portfolio/src/main/webapp/script.js

```diff
@@ -32,7 +32,7 @@ function addRandomGreeting() {
  */
 function addRandomCat() {
   const cats =
-      ["/images/kitten-in-bed.jpg", "/images/sleepy-kitten.jpg"];
+      ["images/kitten-in-bed.jpg", "images/sleepy-kitten.jpg"];
```

Comment:
for actual constants, with the comment below also capitalize them const CATS vs const cats. This will make it easier to differentiate between the `const greetings` which is the actual 'global' constant, vs `const greeting` which is the calculated greeting in the function.